src/Symfony/Component/Notifier/Bridge/LinkedIn/Tests/LinkedInTransportTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Symfony\Component\Notifier\Bridge\LinkedIn\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\Notifier\Bridge\LinkedIn\LinkedInTransport;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+final class LinkedInTransportTest extends TestCase
+{
+    public function testToString(): void
+    {
+        $this->assertSame(sprintf('linkedin://host.test'), (string) $this->getTransport());
+    }
+
+    public function testSupportsChatMessage(): void
+    {
+        $transport = $this->getTransport();
+
+        $this->assertTrue($transport->supports(new ChatMessage('testChatMessage')));
+        $this->assertFalse($transport->supports($this->createMock(MessageInterface::class)));
+    }
+
+    public function testSendNonChatMessageThrows(): void
+    {
+        $this->expectException(LogicException::class);
+
+        $transport = $this->getTransport();
+
+        $transport->send($this->createMock(MessageInterface::class));
+    }
+
+    public function testSendWithEmptyArrayResponseThrows(): void
+    {
+        $this->expectException(TransportException::class);
+
+        $response = $this->createMock(ResponseInterface::class);
+        $response->expects($this->exactly(2))
+            ->method('getStatusCode')
+            ->willReturn(500);
```

Comment:
The same case here, why you didnt use the different const StatusCode in use Symfony\Component\HttpFoundation\Response

Reply:
same as above for common HTTP status code I prefer keep number. 500 is very famous :smiley: